src/tooling/docs-assembler/assembler.yml

```diff
@@ -94,7 +94,7 @@ narrative:
 #  cloud:
```

Comment:
@shainaraskas I think this is the wrong file. @reakaleek we should remove these comments to avoid confusion.

This is defined here: https://github.com/elastic/docs-builder/blob/e55ea1bbadd3db4d23dd632100a7a887cd39557f/src/Elastic.Documentation.Configuration/versions.yml#L26

It looks like we already have 1.15.0 marked as current.

Reply:
We had to move versioning out of `assembler.yml` as it is needed for individual repo builds as well.

Reply:
Aw! Clean up will follow.

Reply:
sorry, my b everyone.

Reply:
Updated in `450b51b`

Reply:
Does 1.13 link anywhere?

Reply:
Discussed on zoom. Fixed in https://github.com/elastic/docs-builder/pull/1511/commits/3a9965ea8920e18e882696b4d355f89600cae2b2